pkg/eigenState/operatorPISplits/operatorPISplits.go

```diff
@@ -0,0 +1,277 @@
+	base.BaseEigenState
+	StateTransitions types.StateTransitions[[]*OperatorPISplit]
+	DB               *gorm.DB
+	Network          config.Network
+	Environment      config.Environment
+	logger           *zap.Logger
+	globalConfig     *config.Config
+
+	// Accumulates state changes for SlotIds, grouped by block number
+	stateAccumulator map[uint64]map[types.SlotID]*OperatorPISplit
+}
+
+func NewOperatorPISplitModel(
+	esm *stateManager.EigenStateManager,
+	grm *gorm.DB,
+	logger *zap.Logger,
+	globalConfig *config.Config,
+) (*OperatorPISplitModel, error) {
+	model := &OperatorPISplitModel{
+		BaseEigenState: base.BaseEigenState{
+			Logger: logger,
+		},
+		DB:               grm,
+		logger:           logger,
+		globalConfig:     globalConfig,
+		stateAccumulator: make(map[uint64]map[types.SlotID]*OperatorPISplit),
+	}
+
+	esm.RegisterState(model, 9)
+	return model, nil
+}
+
+func (ops *OperatorPISplitModel) GetModelName() string {
+	return "OperatorPISplitModel"
+}
+
+type operatorPISplitOutputData struct {
+	ActivatedAt             uint64 `json:"activatedAt"`
+	OldOperatorAVSSplitBips uint64 `json:"oldOperatorAVSSplitBips"`
+	NewOperatorAVSSplitBips uint64 `json:"newOperatorAVSSplitBips"`
+}
+
+func parseOperatorPISplitOutputData(outputDataStr string) (*operatorPISplitOutputData, error) {
+	outputData := &operatorPISplitOutputData{}
+	decoder := json.NewDecoder(strings.NewReader(outputDataStr))
+	decoder.UseNumber()
+
+	err := decoder.Decode(&outputData)
+	if err != nil {
+		return nil, err
+	}
+
+	return outputData, err
+}
+
+func (ops *OperatorPISplitModel) handleOperatorPISplitBipsSetEvent(log *storage.TransactionLog) (*OperatorPISplit, error) {
+	arguments, err := ops.ParseLogArguments(log)
+	if err != nil {
+		return nil, err
+	}
+
+	outputData, err := parseOperatorPISplitOutputData(log.OutputData)
+	if err != nil {
+		return nil, err
+	}
+
+	activatedAt := time.Unix(int64(outputData.ActivatedAt), 0)
+
+	split := &OperatorPISplit{
+		Operator:                strings.ToLower(arguments[1].Value.(string)),
+		ActivatedAt:             &activatedAt,
+		OldOperatorAVSSplitBips: outputData.OldOperatorAVSSplitBips,
+		NewOperatorAVSSplitBips: outputData.NewOperatorAVSSplitBips,
+		BlockNumber:             log.BlockNumber,
+		TransactionHash:         log.TransactionHash,
+		LogIndex:                log.LogIndex,
+	}
+
+	return split, nil
+}
+
+func (ops *OperatorPISplitModel) GetStateTransitions() (types.StateTransitions[*OperatorPISplit], []uint64) {
+	stateChanges := make(types.StateTransitions[*OperatorPISplit])
+
+	stateChanges[0] = func(log *storage.TransactionLog) (*OperatorPISplit, error) {
+		operatorPISplit, err := ops.handleOperatorPISplitBipsSetEvent(log)
+		if err != nil {
+			return nil, err
+		}
+
+		slotId := base.NewSlotID(operatorPISplit.TransactionHash, operatorPISplit.LogIndex)
+
+		_, ok := ops.stateAccumulator[log.BlockNumber][slotId]
+		if ok {
+			fmt.Printf("Submissions: %+v\n", ops.stateAccumulator[log.BlockNumber])
```

Comment:
remove this before merging

Reply:
Done in [1c3161b](https://github.com/Layr-Labs/sidecar/pull/112/commits/1c3161b691271125c3c7f669d3a47b6e015f1e9c)